src/app/services/scheduler/scheduler.service.spec.ts

```diff
@@ -20,25 +20,84 @@ import {HttpClientTestingModule} from '@angular/common/http/testing';
 import {TestBed} from '@angular/core/testing';
 import {EnvconfigService} from '@app/services/envconfig/envconfig.service';
 import {MockEnvconfigService} from '@app/testing/mocks';
-import {configureTestSuite} from 'ng-bullet';
 
 import {SchedulerService} from './scheduler.service';
+import {SchedulerResourceInfo} from '@app/models/resource-info.model';
+import {NOT_AVAILABLE} from '@app/utils/constants';
 
 describe('SchedulerService', () => {
   let service: SchedulerService;
 
-  configureTestSuite(() => {
+  beforeEach(() => {
     TestBed.configureTestingModule({
       imports: [HttpClientTestingModule],
       providers: [SchedulerService, { provide: EnvconfigService, useValue: MockEnvconfigService }],
     });
-  });
-
-  beforeEach(() => {
     service = TestBed.inject(SchedulerService);
   });
 
   it('should create the service', () => {
     expect(service).toBeTruthy();
+    
+  });
+
+  it('should format SchedulerResourceInfo correctly', () => {
+    type TestCase = {
+      description: string;
+      schedulerResourceInfo: SchedulerResourceInfo;
+      expected: string;
+    };
+  
+    const testCases: TestCase[] = [
+      {
+        description: 'test simple resourceInfo',
+        schedulerResourceInfo: {
+          memory: 1024,
+          vcore: 2,
```

Comment:
For consistency should this not be `'memory'` and `'vcore'` like it is for `'pods'` etc.

Reply:
Align to 'memory', 'vcore'. (e52527271f4c703105d7e1c2bac27ffa67bdcd1a)